packages/pangea-sdk/pangea/services/sanitize.py

```diff
@@ -139,6 +218,44 @@ def request_upload_url(
         sha256: Optional[str] = None,
         uploaded_file_name: Optional[str] = None,
     ) -> PangeaResponse[SanitizeResult]:
+        """
+        Sanitize via presigned URL
+
+        Apply file sanitization actions according to specified rules via a
+        [presigned URL](https://pangea.cloud/docs/api/presigned-urls). Beta API.
+
+        OperationId: sanitize_post_v1beta_sanitize 2
+
+        Args:
+            transfer_method: The transfer method used to upload the file data.
+            params: File upload parameters.
+            file_scan: Options for File Scan.
+            content: Options for how the file should be sanitized.
+            share_output: Integration with Secure Share.
+            size: The size (in bytes) of the file. If the upload doesn't match, the call will fail.
+            crc32c: The CRC32C hash of the file data, which will be verified by the server if provided.
+            sha256: The hexadecimal-encoded SHA256 hash of the file data, which will be verified by the server if provided.
+            uploaded_file_name: Name of the user-uploaded file, required for `TransferMethod.PUT_URL` and `TransferMethod.POST_URL`.
+
+        Raises:
+            PangeaAPIException: If an API error happens.
+
+        Returns:
+            The sanitized file and information on the sanitization that was
```

Comment:
In `request_upload_url` we do not return a sanitized file or its information, we should just return presigned url info, then, after upload the file, we should poll the result

Reply:
Corrected.